.teamcity/API/Project.kt

```diff
@@ -18,9 +18,6 @@ object APIProject : Project({
 
     params{
         param("project.directory", "./EdFi.Buzz.Api");
-        param("octopus.release.version","<placeholder value>")
-        param("octopus.release.project", "Buzz API")
-        param("octopus.project.id", "Projects-111")
```

Comment:
No longer trying to deploy out to the server via Octopus?

Reply:
I thought we should be using the unified installer PowerShell in Octopus to download Buzz apps from the Azure Artifact repo and deploy them just as our end users will.

Currently, the deploy.kt for the four Buzz apps is just to publish to the artfact repository, and not to kick off an Octopus project. 

Should I stop using deploy.kt in the Apps (db, api, ui, and etl) except for the EdFi.Buzz.Installer teamcity project?

Reply:
Confirmed with David use the unified installer to deploy in Installer's Deploy.kt build config. 

The remaining Deploy.kt's will be (as here) used from publishing to the artifact repository.